social-team.md

```diff
@@ -0,0 +1,80 @@
+# Social Team
+
+Node.js as a project has a Social Team, a joint team between the TSC and CommComm to which the project has delegated management of official social media.
+
+## Responsibilities
+
+The Social Team is tasked with maintaining the project social accounts. This includes:
+
+- Content creation
+- Amplification of our community
+  - Sharing, retweeting, and so on
+- Engagement with our community
+  - Liking, replying, and so on
+- Updating account metadata (like bio, avatar, header images, and so on)
+
+With regard to posts and community interaction, there is no requirement or entitlement - rather, posts and interactions are completely at the discretion of the Social Team's members.
+
+### Platforms 
+
+This team controls and maintains the various social platforms that Node.js has:
+
+- [Node.js on Twitter](https://twitter.com/nodejs)
+- [Node.js on Facebook](https://www.facebook.com/nodejsfoundation)
+- [Node.js on LinkedIn](https://www.linkedin.com/company/node.js-foundation/)
+
+### Access
+
+Social Team members will have full access to all [platforms](#platforms).
+
+### Expectations
+
+The basic expectations are that **all** Social Team members will have both a willingness and ability to...
+
+- Spend time weekly engaging on various platforms, or communicate the inability to do so with other social team members and work with them to ensure there is *some* coverage
+- Spend time working with other social team members on building out good content
+- Spend time working work project committees, working groups, initiatives, and teams to build effective outward communication as needed
+
+### Repositories
+
+The Social Team will be responsible for and maintain the following:
+
+- [nodejs/tweet](http://github.com/nodejs/tweet): a repo leveraging [Twitter Together](https://github.com/gr2m/twitter-together) that enables project members to queue up tweets to [@nodejs](https://twitter.com/nodejs) from within GitHub.
+
+## Membership
+
+### Nominations
+
+Nominations and self-nominations are open to **project members** on a rolling basis. An issue should be created in [nodejs/admin](https://github.com/nodejs/admin) for any kind of nomination.
```

Comment:
Fix: Readability. "Rolling" may be a colloquialism and not well understood by non-native English speakers.  

```suggestion
Nominations and self-nominations are open to **project members** on an ongoing basis. An issue should be created in [nodejs/admin](https://github.com/nodejs/admin) for any kind of nomination.
```